plasmapy/plasma/equilibria1d.py

```diff
@@ -0,0 +1,106 @@
+"""Functionality for representing one-dimensional equilibria."""
+
+__all__ = ["HarrisSheet"]
+
+import astropy.constants as const
+import astropy.units as u
+import numpy as np
+
+from plasmapy.utils.decorators.validators import validate_quantities
+
+
+class HarrisSheet:
+    """
+    Define a Harris Sheet Equilibrium.
+    Magnetic field will be in the :math:`\pm x` direction and
+    the current density will be in the :math:`\pm z` direction,
+    :math:`\hat{x} \times \hat{y} = \hat{z}` coordinate system.
+
+
+    Parameters
+    ----------
+    B0 : `~astropy.units.Quantity`
+         Magnitude of magnetic field in the limit of :math:`y → ∞` in units
+         convertible to teslas.
+
+    delta : `~astropy.units.Quantity`
+        The thickness of the current sheet in units convertible to meters.
+
+    P0 : `~astropy.units.Quantity`
+        The plasma pressure in the limit of :math:`y → ∞` in units convertible to pascals.
```

Comment:
```suggestion
        The plasma pressure in the limit of :math:`y → ∞` in units
        convertible to pascals.
```
...since there's a convention to have lines in docstrings be ≤ 72 characters.